README.md

```diff
@@ -245,6 +270,47 @@ Used for estimating sizes of macros with long build times and checking if they w
 
 These targets print RAM summaries for a given module.
 
+#### Open and GUI Targets
+
+Those targets are used to run OpenROAD CLI or GUI with Docker flow. They are executable rules, therefore have to be called with `run`:
+
+```
+bazel run L1MetadataArray_full_final_open
+bazel run L1MetadataArray_full_final_gui
+```
+
+CLI and GUI is not available for all stages, consequently these targets are created only for:
+* synthesis
+* floorplan
+* place
+* clock tree synthesis
+* route
+* final
+
+To use them with local flow it is enough to call generated script with `open_{stage}` or gui_{stage}` make target:
+
+```
+# Build dependencies
+bazel build L1MetadataArray_full_final_open_deps
+
+bazel build L1MetadataArray_full_final_make
+./bazel-bin/L1MetadataArray_full_final_make open_final
+# or
+./bazel-bin/L1MetadataArray_full_final_make gui_final
+```
+
+The exception is CLI and GUI for Synth stage, which requires additional LEF files provided with `_open_synth_make` and `_gui_synth_make` target:
+
+```
+bazel build L1MetadataArray_full_open_synth_open_deps
```

Comment:
We want bazel-orfs to be a wafer thin layer on top of ORFS, so refer as much as possible to ORFS and don't create an additional user-facing set of commands in bazel-orfs, unless necessary.

Introduce a single _deps target. This target can be used for gui, open or `make issue`. It should include dependencies for all the cases. When creating an issue, it might be useful to have .lib & .lef files in there.

remove _gui/_open targets and have only _make targets.